manifests/config/exporter/patch/patch-ci.yaml

```diff
@@ -17,3 +17,7 @@ spec:
         hostPath:
           path: /usr/src
           type: Directory
+      - name: proc
+        hostPath:
+          path: /proc-host
```

Comment:
Change the proc-host is not only for the CI.
It is for any deployment in Kind cluster, even in bare-metal

Reply:
Should we change from ci to kind?

Reply:
It would be great!